config/settings/base.py

```diff
@@ -98,6 +98,12 @@
 CSRF_COOKIE_SECURE = False
 SECURE_SSL_REDIRECT = False  # Turn on in production
 REST_FRAMEWORK = {
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.ScopedRateThrottle',
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'rating': '10/day'
```

Comment:
Given that the entire front-end uses one user, raise this a bit ( to 1000 ).

It will see a minor revision ( not in this PR ) to peg it on sessions, rather than users.